### PR TITLE
Issue #4365: Removed getConfigPath

### PR DESCRIPTION
**Pull request by djydewang**
Issue #4365

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java [hunk 1]
```diff
@@ -61,14 +61,10 @@ protected String getPath(String filename) throws IOException {
...
 
-    private String getConfigPath(String filename) throws IOException {
-        return super.getPath("configs" + File.separator + filename);
-    }
-
     @Test
     public void testStaticHeader() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(HeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getConfigPath("java.header"));
+        checkConfig.addAttribute("headerFile", getPath("inputheaderjava.header"));
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java]: `inputheader` => `InputHeader`.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java [hunk 2]
```diff
@@ -129,7 +125,7 @@ public void testNonExistingHeaderFile() throws Exception {
     @Test
     public void testInvalidCharset() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(HeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getConfigPath("java.header"));
+        checkConfig.addAttribute("headerFile", getPath("inputheaderjava.header"));
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java]: same.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java [hunk 3]
```diff
@@ -181,22 +177,22 @@ public void testNullFilename() throws Exception {
     @Test
     public void testNotMatch() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(HeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getConfigPath("java.header"));
+        checkConfig.addAttribute("headerFile", getPath("inputheaderjava.header"));
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java]: same.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java [hunk 4]
```diff
@@ -181,22 +177,22 @@ public void testNullFilename() throws Exception {
...
         checkConfig.addAttribute("ignoreLines", "");
         final String[] expected = {
             "2: " + getCheckMessage(MSG_MISMATCH,
                     "// checkstyle: Checks Java source code for adherence to a set of rules."),
         };
-        verify(checkConfig, getConfigPath("java2.header"), expected);
+        verify(checkConfig, getPath("inputheaderjava2.header"), expected);
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java]: same.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java [hunk 5]
```diff
@@ -181,22 +177,22 @@ public void testNullFilename() throws Exception {
...
 
     @Test
     public void testIgnore() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(HeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getConfigPath("java.header"));
+        checkConfig.addAttribute("headerFile", getPath("inputheaderjava.header"));
         checkConfig.addAttribute("ignoreLines", "2");
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getConfigPath("java2.header"), expected);
+        verify(checkConfig, getPath("inputheaderjava2.header"), expected);
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java]: same.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java [hunk 6]
```diff
@@ -253,7 +249,7 @@ public void testIoExceptionWhenLoadingHeaderFile() throws Exception {
     @Test
     public void testCacheHeaderFile() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(HeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getConfigPath("java.header"));
+        checkConfig.addAttribute("headerFile", getPath("inputheaderjava.header"));
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java]: same.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java [hunk 7]
```diff
@@ -298,9 +294,9 @@ public void testCacheHeaderWithoutFile() throws Exception {
     @Test
     public void testIgnoreLinesSorted() throws Exception {
         final DefaultConfiguration checkConfig = createCheckConfig(HeaderCheck.class);
-        checkConfig.addAttribute("headerFile", getConfigPath("java.header"));
+        checkConfig.addAttribute("headerFile", getPath("inputheaderjava.header"));
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java]: same.

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java [hunk 8]
```diff
@@ -298,9 +294,9 @@ public void testCacheHeaderWithoutFile() throws Exception {
...
         checkConfig.addAttribute("ignoreLines", "4,2,3");
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
-        verify(checkConfig, getConfigPath("java3.header"), expected);
+        verify(checkConfig, getPath("inputheaderjava3.header"), expected);
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java]: same.

Comment (djydewang) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/header/HeaderCheckTest.java]: Fixed it.

**Pull request merged**